packages/core/release-content.txt

```diff
@@ -47,6 +72,7 @@ package/ios/Sources/GlobalState.swift
 package/ios/Sources/QueueHolder.swift
 package/ios/Sources/RNDdSdkConfiguration.swift
 package/lib
+package/lib/TestApplication.kt
```

Comment:
it seems this one doesn't exist?

Reply:
Good catch, it was a file that was moved by mistake and git-ignored. I updated the scripts to make sure the lib folders are removed before building the app